crates/transaction-workloads-lib/src/move_workloads.rs

```diff
@@ -320,7 +327,9 @@ impl EntryPointTrait for EntryPoints {
             | EntryPoints::FungibleAssetMint
             | EntryPoints::APTTransferWithPermissionedSigner
             | EntryPoints::APTTransferWithMasterSigner => "framework_usecases",
-            EntryPoints::OrderBook { .. } => "experimental_usecases",
+            EntryPoints::OrderBook { .. }
+            | EntryPoints::ExistenceCheck { .. }
+            | EntryPoints::CheckAndModify => "experimental_usecases",
```

Comment:
FYI, we can keep this in experimental, but otherwise the orderbook is there because it uses unstable compiler version while the rest of packages use stable version. For your changes stable is also fine. But we can keep as is to avoid further changes